html/about.html

```diff
@@ -0,0 +1,136 @@
+      </div>
+      <div class="about">
+        <h1>We Take Care Of Your Healthy Life.</h1>
+        <div class="about-desc">
+          The cost of an online doctor will vary depending on the telehealth
+          platform and your insurance status. We accept patients with or without
+          insurance. Most patients with in-network insurance pay just $30 to
+          speak with one of our physicians. For self-pay patients, an online
+          doctor appointment costs $129, and follow-ups are just $69. This is
+          less than the typical cost at an in-person clinic.
+        </div>
+        <div class="about-contact">
+          <p>Please contact us per mail or any other questions about Medical</p>
+          <a>ccsubmit2015@cckorea.org</a>
+        </div>
+      </div>
+    </div>
+
+    <div class="section about-logo">
+      <h2 class="title">
+        <span>Medical Logo</span>
+      </h2>
+      <p>
+        the logo of cc ccomons global submit 2015 was decided through the logo
+        competition from 8.june to 7.jully
+      </p>
+      <div class="logo">
+        <img src="../assets/medical-logo.png" alt="logo" />
+      </div>
+    </div>
+
+    <div class="section about-past">
+      <h2 class="title"><span>see our famous department</span></h2>
+      <p>
+        The heart is one of the most important organs of man. Every one knpws
+        that it is he who is under the greatest burden, so it is nott
+        surrprising that with the modern rhytm of life, a huge number of people
+        experience problems with the heart and blood vessels.
+      </p>
+      <div class="content">
+        <div class="last-submit">
+          <img src="../assets/cardio.jpeg" alt="cardio" />
+          <div class="depart">
+            <p class="year">Cardiology</p>
+            <p>
+              the study and treatment of disorders of the heart and the blood
+              vessels.
+            </p>
+          </div>
+        </div>
+        <div class="last-submit">
+          <img src="../assets/plastic.jpeg" alt="plastic" />
+          <div class="depart">
+            <p class="year">Plastic Surgery</p>
+            <p>
+              surgical specialty involving the restoration, reconstruction, or
+              alteration of the human body.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- partners -->
+    <!-- <div class="section partners">
+      <div class="tiitle partner-title"><span>Partner</span></div>
+      <div class="content partner-content">
+        <img src="../assets/mozilla.png" alt="mozilla" />
+        <img src="../assets/google.png" alt="google" />
+        <img src="../assets/naver.png" alt="naver" />
+        <img src="../assets/kakao.png" alt="kakao" />
+        <img src="../assets/airbnb.png" alt="airbnb" />
+      </div>
+    </div> -->
+     <div class="footer">
+        <div class="footer-img">
+<img src="../assets/medical-logo.png" alt="" />
+        </div>
+      
```

Comment:
- [ ] Great job so far but for the mbile version about page I think here you need to add both the partner section and footer to follow the original design so kindly keep it there but in the desktop version you will just keep the last footer with dark background color so please check the design guidlines well 💯🔥💪🏻🚀

- [ ] As I can see also here you need to fix the size since it is too big here in the mobile version so kindly adjust the same size 🚀💪🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/197081694-78a038bc-ceff-4009-8724-d3300a8a4b92.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/197081721-20f5d545-9720-4687-b6ae-157593ea4f35.png)


- Your footer on the desktop I think it is too big as well so kidnly decrease the margin/padding or the height you put for it to not be like this to improve your design to be identical as the given design 📌💯

![image](https://user-images.githubusercontent.com/58553711/197081944-6c7ba472-a3c9-4212-afd6-ae6cb97a2b52.png)

Reply:
@Reem-lab, Thank you for your feedback, i will ensure that all is adjusted to match the design guidelines.